### PR TITLE
✨ respond to stock `$`symbols with basic information

### LINKED ISSUE #50: Respond to stock symbols with basic information
> Human: $SHOP today, bruhhhhhhhhhhhhhh!!!!!
> DuckBot: SHOP.TO: $x per share, +-$x (+-%) today

Can include some other info if it's easy to get. Maybe a link for more info to somewhere.

Should trigger for `$ticker` in all channels, but in the #stonks channel, have it check every word if it's a symbol (in.which case there may be multiple matches; may consider grouping the stocks into a single message to reduce noise).
#42 can be tackled as well.

So, in #stonks 
> Human: shop today bruh, eating amzn's lunch
> DuckBot: SHOP.TO: blah
>                 AMZN: blah

### PR DESCRIPTION
**Pull request by twentylemon**
##### Summary

Fixes https://github.com/duck-dynasty/duckbot/issues/50, mostly. The issue mentions checking every word for posts in the `#stonks` channel, which I simply don't want to do any more. I think it'd be too noisy.

##### Checklist

- [x] this is a source code change
  - [x] run `format` in the repository root
  - [x] run `pytest` in the repository root
  - [x] link to issue being fixed using a [_fixes keyword_](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue), if such an issue exists
  - [x] update the wiki documentation if necessary
- [ ] or, this is **not** a source code change

### CODE REVIEW

File: tests/cogs/fortune/stocks_test.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    message.content = "bruh, no stocks!"
+    clazz = Stocks(bot)
+    await clazz.stock_info(message)
+    message.channel.send.assert_not_called()
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("text", ["$dux", "$DUX me up so I can do the $dux", "I got me some $dux"])
+@mock.patch("yfinance.Ticker")
+async def test_stock_info_single_stock(ticker, bot, message, text):
```
Comment (jameshughes89) [inline tests/cogs/fortune/stocks_test.py]: Also ignores duplicates ---- perhaps add to name?

**Comment by jameshughes89** [comment]
> rick



> rick

We got em'

**Review by jameshughes89 — APPROVED** [review]

**Comment by jameshughes89** [comment]
we waitin' to merge this bad boy?

**Comment by twentylemon** [comment]
Not really. I was waiting for https://github.com/duck-dynasty/duckbot/pull/446 since the service is currently not being updated with new docker images. Keeping the set of changes small in case something did done broke.

**Pull request merged**